### PR TITLE
Bump Broker to 0.4.9

### PR DESCRIPTION
**Pull request by JacobCallahan**
Now that Broker is back on PyPI, let's clean up the requirements

trigger: test-robottelo
pytest: tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -32,9 +30,4 @@ wrapanapi==3.6.0
 # Get airgun, nailgun and upgrade from master
 git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
 git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
```
Comment (JacobCallahan) [inline requirements.txt:32]: newer style dependency definition
```suggestion
airgun @ git+https://github.com/SatelliteQE/airgun.git@master#egg=airgun
nailgun @ git+https://github.com/SatelliteQE/nailgun.git@master#egg=nailgun
```

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'

**Comment by JacobCallahan** [comment]
trigger: test-robottelo
pytest: tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'

**Comment by JacobCallahan** [comment]
Test Failure Analysis for PRT.
23 Executed, 12 Failures, 0 related to the changes in this PR.

| Test                                                                                          | Reason                   |
| --------------------------------------------------------------------------------------------- | ------------------------ |
| tests.foreman.api.test_registration.test_host_registration_end_to_end[rhel9]                  | Bad Checkout             |
| tests.foreman.api.test_registration.test_host_registration_end_to_end[rhel6]                  | Bad Checkout             |
| tests.foreman.api.test_registration.test_host_registration_end_to_end[rhel9_fips]             | Bad Checkout             |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel6]                        | Failed Setup (Container) |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel7]                        | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel7_fips]                   | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel8]                        | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel8_fips]                   | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel9]                        | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_host.test_positive_erratum_applicability[rhel9_fips]                   | Failed Setup (500 ISE)   |
| tests.foreman.cli.test_registration.test_host_registration_end_to_end[rhel6]                  | Bad Checkout             |
| tests.foreman.cli.test_repository.TestRepository.test_positive_upload_content_to_file_repo[0] | Incorrect Test Logic     |

File: requirements.txt [hunk 2]
```diff
@@ -1,16 +1,14 @@
 # Version updates managed by dependabot
 
 betelgeuse==1.11.0
-# broker[docker]==0.4.1 - Temporarily disabled, see below
+broker[docker]==0.4.8
```
Comment (JacobCallahan) [inline requirements.txt:4]:
```suggestion
broker[docker]==0.4.9
```

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'

**Review by pondrejk — APPROVED** [review]
ack, retriggered prt just in case

**Pull request merged**